package.json

```
@@ -50,5 +47,8 @@
     "directory",
     "base",
     "wildcard"
-  ]
+  ],
+  "dependencies": {
+    "is-glob": "^4.0.3"
+  }
```

Looks like this got moved to the bottom. We have a specific ordering of fields, so I'll commit some suggestions to clean that up.
```suggestion
  ]
```